canonical/canonical.com
# [WD-12238] new webpage at /data/opensearch/managed

## Done

Added new webpage according to this [copy doc](https://docs.google.com/document/d/19gcIoKEibXTvZCsmtYCSQBMnBUmNUJI6RaLoMHL2_TA/edit) and this [design](https://www.figma.com/design/uJUwbUAGDog4KCMivrMpnl/24.10-Data-Fabric?node-id=166-4958&t=e5exMaJ2QIAaTFEt-0).

## QA

- Open webpage at https://canonical-com-1317.demos.haus/data/opensearch/managed and check if it complies with the copy doc and the design.

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-12238
